### ISSUE TITLE
storjshare uninstaller does not cleanly uninstall

### ISSUE THREAD

**Issue by computeronix** [Storj/storjshare-gui#537]
When checking the box to remove personal data %appdata%\Storj Share does not get removed

Utilized Storjshare gui winx64 bit installer  (v4 / v5)

**Comment by computeronix** [comment]
chatting with littleskunk and fixing the code here resolves it
https://github.com/Storj/storjshare-gui/blob/master/resources/windows/installer.nsi#L198

change it to:

rmdir /s /q

**Comment by computeronix** [comment]
Registry items too left over in 

"Software\storj share"
"Software\Microsoft\Windows\CurrentVersion\Uninstall\storj share"

**Comment by computeronix** [comment]
C:\Program Files\storj share left over items as well

**Comment by littleskunk** [comment]
The uninstalled removed `%appdata%\name` (storjshare). I changed it to `%appdata%\productName` (Storj Share).

`SetRegView 64` has to be called from the uninstaller as well. That is the reason the unstaller did not remove the registry keys.

One last problem. I am installing the 32bit binary but it is installed like a 64bit applicate. Wrong installation folder and wrong registry keys. I will repair that tomorrow.

**Comment by littleskunk** [comment]
http://forums.winamp.com/showthread.php?t=333931

**Comment by littleskunk** [comment]
At the end I found out that NSIS can detect the 64bit OS but has problems with 32bit binaries on a 64bit OS. I solved that problem by using nodejs. The 32bit binary is now installed in `C:\Program Files (x86)` and the name for uninstaller and start menu entry is `Storj Share (32-bit)`. Both versions (32 and 64 bit) can be installed side by side.

Issue closed. Let me know if I did a mistake and have to reopen the issue.
